### PR TITLE
HTTP packet fragmentation support

### PR DESCRIPTION
**Pull request by jpcofr**

### CODE REVIEW

File: Tests/Packet++Test/PacketExamples/Http1xFrag3.dat
Comment (jpcofr) [inline Tests/Packet++Test/PacketExamples/Http1xFrag3.dat:1]: @seladb Could you explain how these text files are generated and encoded? Do you start by creating traces with a few packets, then make new traces with just one packet from the original trace, and then convert the binary data to hexadecimal?

Comment (seladb) [reply Tests/Packet++Test/PacketExamples/Http1xFrag3.dat:1]: @jpcofr you can use Wireshark for it:
- Open the pcap file in Wireshark
- Choose the packet you want to generate the text file for
- Right click -> Copy -> ... as a Hex Stream
- Paste the result in a text file: `MyPacket.dat`

image

**Comment by seladb** [comment]
@jpcofr are you still working on this PR?

**Comment by jpcofr** [comment]
@seladb  Currently, no. I am planning to continue working on it soon.

**Comment by seladb** [comment]
@jpcofr this PR has been open for a very long time. Maybe we can close it and you can work on it on a separate branch and open another PR once ready?

**Pull request closed without merging**